### PR TITLE
Feature(OGCIO) - 15380 integrate profile

### PR DESCRIPTION
**Pull request by SamSalvatico**
[comment]: <> (This file has been added on OGCIO fork)
### Ticket:

- [15380](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_workitems/edit/15380)

### Description
Added seeding for profile and profile api
## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [x] **New feature**
- [ ] **Dev change**

### CODE REVIEW

**Review by nnorbert — APPROVED** [review]
LGTM!

**Review by edge33 — APPROVED** [review]
LGTM

File: packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json [hunk 1]
```diff
@@ -62,31 +65,51 @@
...
-        "specific_permissions": ["life-events:digital-wallet-flow:*"]
+        "specific_permissions": [
+          "life-events:digital-wallet-flow:*"
+        ]
+      },
+      {
+        "id": "pro-public-servant",
+        "name": "Profile Public Servant",
+        "description": "Profile Public servant",
+        "specific_permissions": [
```
Comment (alfonsograziano) [inline packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json:76]: I was not aware that we have Public servants in the profile! What are they used for?

**Review by alfonsograziano — APPROVED** [review]
LGTM

File: packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json (hunk 1 above)
Comment (SamSalvatico) [reply packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json:76]: Good one, I took it for granted, but it is a really good one

I thought about the possibility that a public servant would be able to manage users entitlements, but I didn't discuss it with anyone.

What do you think about it? Maybe @relvao can help here?

**Pull request merged**